src/shared/components/app/theme.tsx

```diff
@@ -14,45 +15,110 @@ export class Theme extends Component<Props> {
 
     if (hasTheme) {
       return (
-        <Helmet>
+        <>
+          <Helmet>
+            <link
+              rel="stylesheet"
+              type="text/css"
+              href={`/css/themes/${user.local_user_view.local_user.theme}.css`}
+            />
+          </Helmet>
+          <CodeTheme
+            myUserInfo={user}
+            defaultCodeTheme={this.props.defaultCodeTheme}
+          />
+        </>
+      );
+    } else if (
+      this.props.defaultTheme != null &&
+      this.props.defaultTheme != "browser"
+    ) {
+      return (
+        <>
+          <Helmet>
+            <link
+              rel="stylesheet"
+              type="text/css"
+              href={`/css/themes/${this.props.defaultTheme}.css`}
+            />
+          </Helmet>
+          <CodeTheme
+            myUserInfo={user}
+            defaultCodeTheme={this.props.defaultCodeTheme}
+          />
+        </>
+      );
+    } else {
+      return (
+        <>
+          <Helmet>
+            <link
+              rel="stylesheet"
+              type="text/css"
+              href="/css/themes/litely.css"
+              id="default-light"
+              media="(prefers-color-scheme: light)"
+            />
+            <link
+              rel="stylesheet"
+              type="text/css"
+              href="/css/themes/darkly.css"
+              id="default-dark"
+              media="(prefers-color-scheme: no-preference), (prefers-color-scheme: dark)"
+            />
+          </Helmet>
+          <CodeTheme
```

Comment:
There are three cases for themes: `user-selected`, `default`, and `browser`.
There are the same choices, for code themes, so I split the code into its own component.

The question is: should that component be embedded in Theme or be a peer?
I'm leaning to it being a peer, but Ill defer to yo.

Reply:
Splitting it into its own component is way overkill, **exactly because** it uses the same three options as theme. 

Just add an extra one of these to each of the 3 cases which already have blocks above.

```
<link
              rel="stylesheet"
              type="text/css"
              href={`location_of_hljs_theme`}
            />
```

Reply:
Its the same options, but I wanted them to be independent, so there would be 9 possible combos.